baselines/fedavgm/pyproject.toml

```diff
@@ -0,0 +1,139 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "fedavgm" # <----- Ensure it matches the name of your baseline directory containing all the source code
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
```

Comment:
Maybe you want to add your contact details here? also please add the title of the baseline to the `description` field above.

Reply:
Updated `pyproject.toml` with the recommended changes.